torch/_torch_docs.py

```diff
@@ -3368,24 +3368,35 @@ def merge_dicts(*dicts):
 This is a low-level function for calling LAPACK directly. This function
 returns a namedtuple (a, tau) as defined in `LAPACK documentation for geqrf`_ .
 
-You'll generally want to use :func:`torch.qr` instead.
-
-Computes a QR decomposition of :attr:`input`, but without constructing
-:math:`Q` and :math:`R` as explicit separate matrices.
-
-Rather, this directly calls the underlying LAPACK function `?geqrf`
+Computes a QR decomposition of :attr:`input`.
+Both `Q` and `R` matrices are stored in the same tensor `a`.
+The elements of `R` are stored on and above the diagonal.
+Elementary reflectors (or Householder vectors) implicitly defining matrix `Q`
+are stored below the diagonal.
+Result of this function can be used together with :func:`torch.linalg.householder_product`
+to obtain the `Q` matrix or
+with :func:`torch.ormqr` that uses implicit representation of matrix `Q` for efficient matrix-matrix multiplication.
+
+This function directly calls the underlying LAPACK function `geqrf`
 which produces a sequence of 'elementary reflectors'.
 
 See `LAPACK documentation for geqrf`_ for further details.
 
+.. note::
+    To obtain explicit Q and R matrices it is recommended to use :func:`torch.linalg.qr`.
+
+.. note::
```

Comment:
I would combine this note with the previous to make one "see also" note.

Reply:
"See also :func:\`torch.linalg.qr\`, which computes explicit Q and R matrices, and :func:\`torch.linalg.lstsq\` with the \`\`driver="gels"\`\` option for a function that can solve matrix equations using a QR decomposition." ?